workflow/snakemake_rules/nextclade.smk

```diff
@@ -156,31 +154,17 @@ rule combine_alignments:
     shell:
         """
         if [[ -s {input.old_alignment} ]]; then
-            cat {input.old_alignment} {input.new_alignment} > {output.alignment}
+            mv {input.old_alignment} {output.alignment}
+            cat {input.new_alignment} >> {output.alignment}
         else
-            cp {input.new_alignment} {output.alignment}
+            mv {input.new_alignment} {output.alignment}
```

Comment:
Copy of a 300GB file takes 30 minutes - since we don't use the old file anywhere else we can just move, or move and cat whatever needs to be added.
@ivan-aksamentov